lib/mongo/client.rb

```diff
@@ -856,6 +855,19 @@ def default_options(options)
       end
     end
 
+    # Provides some default encryption options and sets up data necessary
+    # for auto-encryption
+    def set_auto_encryption_options
+      opts_copy = @options[:auto_encryption_options].dup
+
+      opts_copy.tap do |opts|
```

Comment:
The tap here is no longer needed, it can be removed and the two nested lines deindented.